lib/commentOnOpenedPullRequest.js

```diff
@@ -17,28 +17,29 @@ var thirdPartyMessage1 = 'I noticed that a file in ';
 var thirdPartyMessage2 = ' has been added or modified. ' +
     'Please verify that it has a section in [LICENSE.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/LICENSE.md) ' +
     'and that its license information is up to date with this new version.  Once you do, please confirm by commenting on this pull request.\n\n';
+var claMessage = 'Please follow the instructions [here](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) ' +
```

Comment:
Nevermind, I see the other part of the message further down.  It would be a lot better if the complete message was in one place.  Maybe you can work on the handlebars next?